jenkins-agents/jenkins-agent-ansible/Dockerfile

```diff
@@ -22,25 +21,22 @@ ARG PIP_PKGS="\
         ansible==${ANSIBLE_VERSION} \
 "
 ARG YUM_PKGS="\
```

Comment:
nit: since you've changed it to dnf, should this arg be updated?

Reply:
Fixed, also changed any use of `yum` to `dnf`.